docs/DeveloperGuide.md

```diff
@@ -257,29 +257,56 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* teaching assistant for one or more modules
+* has a need to manage details of a significant number of students, including:
+  * contact details
+  * attendance
+  * grades
```

Comment:
I think would be good to specify submissions and class part as well

Reply:
Updated!